### ISSUE TITLE
Abnormality of fetal nasal bone

### ISSUE THREAD

**Issue by BimalChaudhari** [obophenotype/human-phenotype-ontology#6989]
**Preferred term label:**
Abnormality of fetal nasal bone

**Synonyms**

**Definition (free text, please give PubMed ID)**
Fetal nasal bone length < 2.5 percentile or absent (PMID:16100637).

**Parent term (use hpo.jax.org/app)**
HP:0011425

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-0115-949X

**Comment by pnrobinson** [comment]
@pnrobinson  test

**Comment by patt1morgan** [comment]
@Shagun1976, @fcradio, @Green-eve. This is the last of the new term requests for 'nasal bone'. Once again, we ask your help in reviewing.  Your help is truly appreciated...

**Comment by Shagun1976** [comment]
Agree with the parent term: Abnormality of fetal nasal bone
Two child terms can be included: Absent nasal bone and hypoplastic nasal bone
Both terms need to be qualified further by 1. Definition 2. Synonyms  
Absent nasal bone
Definition: Non visualization of nasal bone on antenatal ultrasound using standard imaging techniques
Synonyms: Unossified nasal bone
Disease association: Down syndrome (# 190685)
Comment: Absent nasal bone on antenatal ultrasound is one of the soft markers for Down syndrome. This marker can be used to adjust a-priori risk for Down syndrome and offer options for further diagnostic testing to the pregnant woman. Absent nasal bone can also rarely be a marker for submicroscopic copy number abnormalities and single gene disorders in the fetus. 

Hypoplastic nasal bone
Definition: Length of the nasal bone below a predetermined cut-off as seen on antenatal ultrasound using standard imaging 
techniques 
Synonyms: Nasal bone length <5th centile, Nasal bone length <2.5th centile, Nasal bone length <2.5mm, Nasal bone MoM<0.75
Comment: Different studies have used different cut-offs to assess the length of nasal bone. Most data supports use of gestation and ethnicity specific centiles or MoMs. Hypoplastic nasal bone  on antenatal ultrasound is one of the soft markers for Down syndrome. This marker can be used to adjust a-priori risk for Down syndrome and offer options for further diagnostic testing to the pregnant woman. Hypoplastic nasal bone can also rarely be a marker for submicroscopic copy number abnormalities and single gene disorders in the fetus.

**Comment by pnrobinson** [comment]
adding terms
